content/videos/challenges/16-l-system-fractal-trees/index.json

```diff
@@ -5,7 +5,7 @@
   "videoId": "E1B4UoSQMFw",
   "date": "2016-05-31",
   "languages": ["JavaScript", "p5.js", "Processing"],
-  "topics": ["For beginners"],
+  "topics": ["Fractals, Algorithmic Botany"],
```

Comment:
I think it should be `["Fractals", "Algorithmic Botany"]` (two different strings)